test/org/sagebionetworks/bridge/models/surveys/SurveyQuestionOptionTest.java

```diff
@@ -0,0 +1,48 @@
+package org.sagebionetworks.bridge.models.surveys;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import nl.jqno.equalsverifier.EqualsVerifier;
+import org.junit.Test;
+
+public class SurveyQuestionOptionTest {
+    private static final Image DUMMY_IMAGE = new Image("dummy-source", 42, 42);
+
+    @Test
+    public void allValues() {
+        SurveyQuestionOption option = new SurveyQuestionOption("test-label", "test-detail", "test-value", DUMMY_IMAGE);
+        assertEquals("test-label", option.getLabel());
+        assertEquals("test-detail", option.getDetail());
+        assertEquals("test-value", option.getValue());
+        assertEquals(DUMMY_IMAGE, option.getImage());
+
+        String optionString = option.toString();
+        assertTrue(optionString.contains(option.getLabel()));
+        assertTrue(optionString.contains(option.getDetail()));
+        assertTrue(optionString.contains(option.getValue()));
+        assertTrue(optionString.contains(option.getImage().toString()));
+    }
```

Comment:
As per IM conversation, the goal is to test the changes made in the code review. Exhaustively testing SurveyQuestionOption is beyond the scope of this change.